www/components/Nav/Annoucement.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { IconArrowRight } from '@supabase/ui'
+
+import data from '~/data/Annoucement.json';
+
+type A = {
+  show: boolean;
+  text: string;
+  link: {
+    text: string;
+    url: string;
+  }
+}
+
+const a: A = data;
+
+const Annoucement = () => {
+  if (!a.show) return (<></>)
```

Comment:
you might be able to just return `null` here, although it might cause a typescript error